app/uk/gov/hmrc/cataloguefrontend/servicecommissioningstatus/ServiceCommissioningStatusController.scala

```diff
@@ -89,18 +89,33 @@ class ServiceCommissioningStatusController @Inject() (
                                 allChecks <- serviceCommissioningStatusConnector.allChecks()
                                 results   <- serviceCommissioningStatusConnector.cachedCommissioningStatus(formObject.teamName, formObject.serviceType)
                                 checks    =  if (formObject.checks.isEmpty) allChecks.map(_._1).toList else formObject.checks
-
+                                
+                                filteredResults = results.filter { result => //exclude decommissioned services
+                                  result.checks.exists {
+                                    case check: Check.SimpleCheck =>
+                                      check.checkResult match {
+                                        case Right(_) => true
+                                        case _ => false
+                                      }
+                                    case check: Check.EnvCheck =>
+                                      check.checkResults.values.exists {
+                                        case Right(_) => true
+                                        case _ => false
```

Comment:
Minor refactor but you may be able to achieve this with '.isRight'

Reply:
👍